### PR TITLE
main->feature/rds-phase-2 at cd0810fd99f48150bff863b07d5edcb645f6667a

### PR DESCRIPTION
**Pull request by Jkovarik**
##  Summary

* main->feature/rds-phase-2 at cd0810fd99f48150bff863b07d5edcb645f6667a

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [x] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -434,9 +465,6 @@ aws lambda invoke --function-name $PREFIX-data-migration1 \
 
 ### Changed
 
-- **CUMULUS-NONE**
```
Comment (Jkovarik) [inline CHANGELOG.md:437]: This was removed as a dupe in the main branch as part of the 10.1.1 release updates.

File: example/spec/parallel/queueGranules/queueGranulesPostProcessingSpec.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+      expect(queuedRecord.status).toEqual('completed');
+      expect(childWorkflowRecord.status).toEqual('completed');
+    });
+  });
+
+  it('the granule is added to the database by the child workflow', async () => {
+    if (beforeAllFailed) fail('beforeAll() failed');
+    await Promise.all(
+      inputPayload.granules.map(async (granule) => {
+        const record = await waitForModelStatus(
```
Comment (Jkovarik) [inline example/spec/parallel/queueGranules/queueGranulesPostProcessingSpec.js:209]: This test needs to be upgraded to not use the Dynamo model check/etc.

Comment (Jkovarik) [reply example/spec/parallel/queueGranules/queueGranulesPostProcessingSpec.js]: a5fd5f3d1c

File: example/spec/serial/ingestPdrWithNodeNameFtpSpec.js [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+              expect(lambdaOutput.payload.granules.length).toEqual(1);
+              expect(lambdaOutput.payload.pdr).toEqual(lambdaOutput.payload.pdr);
+            }
+          });
+        });
+      });
+    });
+
+    describe('the reporting lambda has received the cloudwatch stepfunction event and', () => {
+      it('the execution record is added to DynamoDB', async () => {
```
Comment (botanical) [inline example/spec/serial/ingestPdrWithNodeNameFtpSpec.js:399]:
```suggestion
      it('the execution record is added to PostgreSQL', async () => {
```

File: example/spec/serial/ingestPdrWithNodeNameFtpSpec.js [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+          const record = await waitForApiStatus(
+            getExecution,
+            { prefix: config.stackName, arn: parsePdrExecutionArn },
+            'completed'
+          );
+          expect(record.status).toEqual('completed');
+        }
+      });
+
+      it('the pdr record is added to DynamoDB', async () => {
```
Comment (botanical) [inline example/spec/serial/ingestPdrWithNodeNameFtpSpec.js:411]:
```suggestion
      it('the pdr record is added to PostgreSQL', async () => {
```

**Review by botanical — APPROVED** [review]

**Pull request merged**